core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4728,6 +4729,27 @@ public String toString() {
   //
   // FUSE integration related properties
   //
+  public static final PropertyKey FUSE_AUTH_POLICY_CLASS =
+      new Builder(Name.FUSE_AUTH_POLICY_CLASS)
+          .setDefaultValue("alluxio.fuse.auth.DefaultAuthPolicy")
+          .setDescription("The fuse auth policy class.")
```

Comment:
Can you list out the available auth policy here?

Reply:
Done.